src/common/payment.ts

```diff
@@ -182,22 +184,44 @@ export class TurboUnauthenticatedPaymentService
     return addresses;
   }
 
+  protected appendCallbackUrlsToQuery(
+    callbackUrls:
+      | TurboCheckoutSessionHostedParams
+      | TurboCheckoutSessionEmbeddedParams,
+  ): string {
+    return Object.entries(callbackUrls)
+      .filter(([_, value]) => value !== undefined)
+      .map(([key, value]) => `${key}=${encodeURIComponent(value)}`)
+      .join('&');
+  }
+
```

Comment:
doesn't appear to be used

```suggestion
```

Reply:
ah this was legacy before using search params - will remove

Reply:
[59c80d6](https://github.com/ardriveapp/turbo-sdk/pull/269/commits/59c80d6299b9c59dd919a25766d285542329be8c)